src/02-common-principles.md

```diff
@@ -486,7 +486,8 @@ and a guide for using macros can be found at
         "Units": "RECOMMENDED",
         "TermURL": "RECOMMENDED",
         "HED": "OPTIONAL",
-   }
+   },
+   "."
```

Comment:
"." feels like it should be equivalent to `None`, and could be left off. Is there a reason to make it explicit in all cases? And is it always `"."`, `".."` or `"../.."` (etc.)? If so, would it make sense to have a kind of depth or level parameter that would translate into number of parent directories to follow?

Finally, is there any chance that the macro calling code would have access to the file from which it's called? If so, we could actually determine this automatically based on the filename.

Reply:
Is this what you're thinking @effigies ?
```
{
    "Depth": "."
}
```

I agree that an empty/null and/or default value should evaluate to `"."` though as that's the sop for just about everything out there.  Is there an issue with just giving this a parameter/key as effigies suggested?

Reply:
implemented in https://github.com/bids-standard/bids-specification/pull/1056/commits/fe3dd0e4bdc19aed3b957fdc52f17e4960b3526a and the 5 following commits